npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cloudgov-dashboard",
-  "version": "1.14.0",
```

Comment:
Good catch, it's supposed to match package.json, so it was an oversight that it wasn't updated to 1.16.0 previously.